src/__mocks__/definitions.js

```diff
@@ -326,6 +326,11 @@ const definitions = [
     maximum: 32,
     exclusiveMaximum: true,
   },
+  {
```

Comment:
This was just a test rigth?

Reply:
It is test only. But in order to avoid confusion, I removed it and used another UCA already in the definitions with the same scenario.